src/components/Dropdowns/StyledSelect.tsx

```diff
@@ -53,6 +55,11 @@ const SelectContainer = styled.div<{ $type: ButtonProps["type"] | "outlined" }>`
 
             &:not(.react-select__control--is-disabled) {
               color: var(--color-text-button);
+
+              & .react-select__single-value,
+              & .react-select__input-container {
+                color: var(--color-text-button);
+              }
```

Comment:
Fixed a bug where dropdowns with the purple primary button style would have the wrong text color when disabled.